app/views/users/show_post.html.erb

```diff
@@ -0,0 +1 @@
+here is the user <%= params[:id]%> post with id <%= params[:post_id]%>
```

Comment:
- [ ]  Kindly make sure to store this file in # app/views/posts/show.html.erb and rename it as show.html.erb